src/pages/en/guides/markdown-content.mdx

```diff
@@ -497,6 +498,12 @@ You can add frontmatter properties to all of your Markdown and MDX files by usin
     }
     ```
 
+    :::tip
+    <Since v="2.0.0" />
+
+    `data.astro.frontmatter` contains all properties from a given Markdown or MDX document. This allows you to modify existing frontmatter properties, or compute new properties based on other properties.
```

Comment:
nit: is there a better description of "other" properties?

Do you mean exactly the file's existing frontmatter properties?

If so, I could see:
> based on these existing properties
> based on these current properties
> compute new properties programmatically from the existing properties

If you truly mean, from "all other random things in the universe" then I might be tempted to just say "or compute new properties." and leave it at that, to avoid saying properties yet again.  😅

Reply:
How many properties could a prop shop prop if a prop shop could prop properties?
...okay "or compute new properties" is fine 😆